hibernate/cache-commons/src/main/java/org/infinispan/hibernate/cache/commons/access/InvalidationCacheAccessDelegate.java

```diff
@@ -123,16 +123,10 @@ public boolean putFromLoad(Object session, Object key, Object value, long txTime
 
 	@Override
 	public void remove(Object session, Object key) throws CacheException {
-		putValidator.setCurrentSession(session);
-		try {
-			// We update whether or not the region is valid. Other nodes
-			// may have already restored the region so they need to
-			// be informed of the change.
-			writeCache.remove(key);
-		}
-		finally {
-			putValidator.resetCurrentSession();
-		}
+		// We update whether or not the region is valid. Other nodes
```

Comment:
aren't you missing the `session`?

Reply:
No, this gets overridden in NonTxInvalidationCacheAccessDelegate. Transactional variant does not need the session.